sql-ballerina/tests/call-procedures-test.bal

```diff
@@ -0,0 +1,393 @@
+    test:assertEquals(paraBigInt.get(int), 9223372036854774807, "3rd out parameter of procedure did not match.");
+    test:assertEquals(paraSmallInt.get(int), 32767, "4th out parameter of procedure did not match.");
+    test:assertEquals(paraTinyInt.get(int), 127, "5th out parameter of procedure did not match.");
+    test:assertEquals(paraBit.get(boolean), true, "6th out parameter of procedure did not match.");
+    test:assertEquals(paraDecimal.get(decimal), paraDecimalVal, "7th out parameter of procedure did not match.");
+    test:assertEquals(paraNumeric.get(decimal), paraDecimalVal, "8th out parameter of procedure did not match.");
+    test:assertTrue((checkpanic paraFloat.get(float)) > 1234.0, "9th out parameter of procedure did not match.");
+    test:assertTrue((checkpanic paraReal.get(float)) > 1234.0, "10th out parameter of procedure did not match.");
+    test:assertEquals(paraDouble.get(float), 1234.56, "11th out parameter of procedure did not match.");
+}
+
+
+
+
+
+@test:Config {
+    groups: ["procedures"]
+}
+function testCreateProcedures1() {
+    MockClient dbClient = checkpanic new (url = proceduresDB, user = user, password = password);
+    ParameterizedQuery createProcedure = `
+        CREATE PROCEDURE InsertStringData(IN p_id INTEGER,
+                                  IN p_varchar_type VARCHAR(255),
+                                  IN p_charmax_type CHAR(10),
+                                  IN p_char_type CHAR,
+                                  IN p_charactermax_type CHARACTER(10),
+                                  IN p_character_type CHARACTER,
+                                  IN p_nvarcharmax_type NVARCHAR(255))
+       MODIFIES SQL DATA
+              INSERT INTO StringTypes(id, varchar_type, charmax_type, char_type, charactermax_type, character_type, nvarcharmax_type)
+              VALUES (p_id, p_varchar_type, p_charmax_type, p_char_type, p_charactermax_type, p_character_type, p_nvarcharmax_type);
+    `;
+    ExecutionResult result = checkpanic dbClient->execute(createProcedure);
+    checkpanic dbClient.close();
+    test:assertExactEquals(result.affectedRowCount, 0, "Affected row count is different.");
+    test:assertExactEquals(result.lastInsertId, (), "Last Insert Id is not nil.");
+}
+
+@test:Config {
+    groups: ["procedures"]
+}
+function testCreateProcedures2() {
+    MockClient dbClient = checkpanic new (url = proceduresDB, user = user, password = password);
+    ParameterizedQuery createProcedure = `
+        CREATE PROCEDURE SelectStringDataWithOutParams (IN p_id INT, OUT p_varchar_type VARCHAR(255),
+                                                OUT p_charmax_type CHAR(10), OUT p_char_type CHAR, OUT p_charactermax_type CHARACTER(10),
+                                                OUT p_character_type CHARACTER, OUT p_nvarcharmax_type NVARCHAR(255))
+            READS SQL DATA DYNAMIC RESULT SETS 2
```

Comment:
Is this needed to run some tests? If so move it to the initialisation SQL query script

Reply:
I tried it first. But it gives me a error. I will try it again

Reply:
As discussed offline lets fix it in latter PR